streaming/java/streaming-runtime/src/main/java/org/ray/streaming/runtime/core/graph/executiongraph/ExecutionJobVertex.java

```diff
@@ -0,0 +1,174 @@
+package org.ray.streaming.runtime.core.graph.executiongraph;
+
+import com.google.common.base.Preconditions;
+import java.util.ArrayList;
+import java.util.HashMap;
+import java.util.List;
+import java.util.Map;
+import org.aeonbits.owner.ConfigCache;
+import org.ray.api.RayActor;
+import org.ray.streaming.jobgraph.JobVertex;
+import org.ray.streaming.jobgraph.LanguageType;
+import org.ray.streaming.jobgraph.VertexType;
+import org.ray.streaming.operator.StreamOperator;
+import org.ray.streaming.runtime.config.master.ResourceConfig;
+import org.ray.streaming.runtime.core.processor.ProcessBuilder;
+import org.ray.streaming.runtime.core.processor.StreamProcessor;
+import org.ray.streaming.runtime.worker.JobWorker;
+
+/**
+ * Physical job vertex which including parallelism execution vertex.
+ */
+public class ExecutionJobVertex {
+
+  private final String jobName;
+  private final int jobVertexId;
+  private final String jobVertexName;
+  private final JobVertex jobVertex;
+  private final StreamProcessor streamProcessor;
```

Comment:
We put `StreamOperator` in graph now

Reply:
Use ```Processor``` in physical plan.  
And the ```Processor``` is already implemented in runtime package.

Reply:
`Processor` is built in JobWorker, not in physical plan. Please see cross-lang job graph

Reply:
Left stream operator here if so.